lib/src/scrollbar95.dart

```diff
@@ -0,0 +1,383 @@
+import 'dart:async';
+import 'dart:math';
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter95/flutter95.dart';
+
+class Scrollbar95 extends StatefulWidget {
+  final Widget child;
+  final double scrollAmount;
+  final double minThumbHeight;
+  final double scrollbarThickness;
+
+  const Scrollbar95({
+    super.key,
+    required this.child,
+    this.scrollAmount = 0.05,
+    this.scrollbarThickness = 30,
+    this.minThumbHeight = 10,
+  }) : assert(scrollAmount > 0 && scrollAmount <= 1);
+
+  @override
+  State<Scrollbar95> createState() => _Scrollbar95State();
+}
+
+class _Scrollbar95State extends State<Scrollbar95> {
+  final ValueNotifier<double> _relativeTrackPosition =
+      ValueNotifier<double>(0.0);
+  final Duration _scrollingInterval = const Duration(milliseconds: 200);
+  final GlobalKey _trackKey = GlobalKey();
+  ScrollController? _controller;
+  Timer? _continuousScrollingTimer;
+  double _thumbExtent = 30;
+  Axis _scrollDirection = Axis.vertical;
+
+  @override
+  Widget build(BuildContext context) {
+    _findControllerAndListen(context);
+
+    IconData scrollBackIcon;
+    IconData scrollAheadIcon;
+
+    switch (_scrollDirection) {
+      case Axis.horizontal:
+        scrollBackIcon = Icons.arrow_left;
+        scrollAheadIcon = Icons.arrow_right;
+        break;
+      case Axis.vertical:
+        scrollBackIcon = Icons.arrow_drop_up;
+        scrollAheadIcon = Icons.arrow_drop_down;
+        break;
+    }
+
+    Widget scrollBackButton = GestureDetector(
+      onTap: _scrollBackOnce,
+      onLongPress: () => _startContinuousScrolling(false),
+      onLongPressEnd: (_) => _stopContinuousScrolling(),
+      child: AspectRatio(
+        aspectRatio: 1,
+        child: Elevation95(
+          child: Transform.scale(
+            scale: 1.5,
+            child: Icon(scrollBackIcon),
+          ),
+        ),
+      ),
+    );
+
+    Widget scrollAheadButton = GestureDetector(
+      onTap: _scrollAheadOnce,
+      onLongPress: () => _startContinuousScrolling(true),
+      onLongPressEnd: (_) => _stopContinuousScrolling(),
```

Comment:
Probably instead of using `GestureDetector` directly this could be implemented using `Button95` (with some modifications) so you can take the advantage of the different container decorations for the tapped state (i.e. pressed). Same for the scrollbar thumb.